README.md

```diff
@@ -25,12 +25,18 @@
 
 ## :arrow_forward: Usage
 
-> `react-native@0.60` or higher
+> `react-native@0.61.0` or higher
 
 ```sh
 npx react-native init MyApp --template react-native-template-typescript
 ```
 
+> `react-native@0.60.x`
+
+```sh
+npx react-native@0.60.6 init MyApp --template react-native-template-typescript@6.2.0
```

Comment:
Is `react-native@0.60.6` really needed? Because the RN version is defined in the `package.json` of the template. Maybe we should also adjust the versioning pattern of the template for the future, otherwise it will get pretty confusing which version of the template is related to which RN version, but that's another discussion.

Reply:
Ok, right I forgot about this. Yeah, I guess we have to rethink versioning.